python/hidet/utils/benchmark/bench.py

```diff
@@ -0,0 +1,221 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List, Optional, Callable, Tuple, Any, Dict, Union
+import time
+from dataclasses import dataclass
+
+import numpy as np
+import torch
+from matplotlib import pyplot as plt
```

Comment:
I am trying to avoid the direct dependency of hidet on pytorch, because there might be other users of hidet (like people use onnx).

Now, hidet will directly import `hidet.utils`, thus, when people import hidet, the torch would be imported.
Consider moving the `import torch` and `import matplotlib` inside the function that used it. So that people do not need to install pytorch to `import hidet`.